pkg/util/util.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import (
+	"fmt"
+	"github.com/code-ready/crc/pkg/crc/output"
+	"strings"
+)
+
+func PromptUserForYesOrNo(message string) bool {
+	var response string
+	output.OutF(message + "? [Y/N]: ")
```

Comment:
I would use "[y/N]" as the default behaviour is to consider the user answered 'no'.

Reply:
right, I agree. This was also the behaviour as in Minishift.

Reply:
Will use `[y/N]`